src/test/munit/derby/connection-derby-test-case.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:munit="http://www.mulesoft.org/schema/mule/munit"
+      xmlns:mtf="http://www.mulesoft.org/schema/mule/mtf"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+        http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
+
+
+		http://www.mulesoft.org/schema/mule/mtf http://www.mulesoft.org/schema/mule/mtf/current/mule-mtf.xsd">
+
+    <munit:config name="connection-derby-test-case.xml" minMuleVersion="4.4.0">
+        <munit:parameterizations>
+            <munit:parameterization name="derby-config">
+                <munit:parameters>
+                    <munit:parameter propertyName="config" value="derby-config"/>
+                    <munit:parameter propertyName="expectedErrorType" value=""/>
+                </munit:parameters>
+            </munit:parameterization>
+        </munit:parameterizations>
+    </munit:config>
+
+    <munit:test name="connection-test" expectedErrorType="${expectedErrorType}">
+        <munit:execution>
```

Comment:
Is it necessary to set an expectedErroType if the expected type is an empty string? Could this attribute be avoided?

Reply:
It's necessary to parameterize the test like that. If it isn't set, it will fail due to not finding the value of that property.